support/oidc-discovery-provider/workload_api.go

```diff
@@ -0,0 +1,167 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"sync"
+	"time"
+
+	"github.com/andres-erbsen/clock"
+	"github.com/sirupsen/logrus"
+	workload_pb "github.com/spiffe/go-spiffe/proto/spiffe/workload"
+	"github.com/spiffe/go-spiffe/workload"
+	"github.com/spiffe/spire/pkg/common/idutil"
+	"github.com/zeebo/errs"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+	"gopkg.in/square/go-jose.v2"
+)
+
+const (
+	DefaultWorkloadAPIPollInterval = time.Second * 5
```

Comment:
The documented default for this is 10 seconds
```suggestion
	DefaultWorkloadAPIPollInterval = time.Second * 10
```